_typography.scss

```diff
@@ -0,0 +1,32 @@
+h1 {
+    font-size: 50px;
+    font-weight: 700;
+}
+h2 {
+    font-size: 40px;
+    font-weight: 700;
+}
+h3 {
+    font-size: 20px;
+    font-weight: 500;
+}
+h4 {
+    font-size: 20px;
+    font-weight: 400;
+}
+h5 {
```

Comment:
h5 font size should be less than h4 or high-level heading elements.

Reply:
it has been changed and resolved

Reply:
it has been changed and resolved